pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -64,21 +63,21 @@ export function showEndGameImage(array: GameBlockState[]) {
   let playerTwoArray = longestSubarray(array, BlockState.PLAYER_TWO_SELECTED);
   console.log("P2", playerTwoArray);
   if (playerOneArray > playerTwoArray) {
-    return <img src="/images/math1/longestStreak/playerOneWinner.png" />;
+    return <img src="/images/math1/longestStreak/playerOneWinner.jpg" />;
   } else if (playerTwoArray > playerOneArray) {
-    return <img src="/images/math1/longestStreak/playerTwoWinner.png" />;
+    return <img src="/images/math1/longestStreak/playerTwoWinner.jpg" />;
   } else if (playerOneArray === playerTwoArray) {
     return <img src="/images/math1/longestStreak/drawWinner.png" />;
   }
 }
 
 export default function BlockComponentGallery() {
   const dispatch = useDispatch();
-  const { stage, blocks: gameState } = useSelector(longestStreakSelector);
-
-  //const [stage, setStage] = useState(STAGE.SET_RULES);
-
-  const [playerOneName, setPlayerOneName] = useState("Player 1");
+  const {
+    stage,
+    blocks: gameState,
```

Comment:
Do you understand what's happening here? This is called json destructing

Reply:
I understand that it's tying the component to the state, doing the getting and the setting and tying it to the payload action.  That's just a more succinct way to do it?